### DISCUSSION TITLE
Show accepted payment methods in POI details

### DISCUSSION THREAD

**Discussion by UgnilJoZ** [organicmaps/organicmaps#812] [Ideas]
I would enjoy if Organic Maps could display the [payment methods](https://wiki.openstreetmap.org/wiki/Key:payment) that a POI accepts. When visiting places like a restaurant, paying by card/contactless is not possible everywhere in my country. So this information would be very handy to see when tapping on a place.

**Comment by biodranik** [comment]
If this information is present/can be added into the OpenStreetMap, then it will be possible to display it in Organic Maps.

**Reply by UgnilJoZ** [reply]
This is how Osmand displays it:
![Screenshot_20210708-111117~2](https://user-images.githubusercontent.com/7910828/124896118-7514dc80-dfdd-11eb-9d9d-0e80fdda85ff.png)
Something in that direction would be really nice for Organic Maps.

**Reply by pred2k** [reply]
Showing the commonly used payment type **icons** would be nicer and more compact.
